api/resource/resource.go

```diff
@@ -31,18 +31,29 @@ type Resource struct {
 }
 
 const (
-	buildAnnotationOriginalName      = konfig.ConfigAnnoDomain + "/originalName"
+	buildAnnotationPreviousName      = konfig.ConfigAnnoDomain + "/originalName"
 	buildAnnotationPrefixes          = konfig.ConfigAnnoDomain + "/prefixes"
 	buildAnnotationSuffixes          = konfig.ConfigAnnoDomain + "/suffixes"
-	buildAnnotationOriginalNamespace = konfig.ConfigAnnoDomain + "/originalNs"
+	buildAnnotationPreviousNamespace = konfig.ConfigAnnoDomain + "/originalNs"
```

Comment:
The actual annotation should change too. Also, these are plural as they contain CSV lists, so
```
buildAnnotationPreviousNames      = konfig.ConfigAnnoDomain + "/previousNames"
buildAnnotationPreviousNamespaces = konfig.ConfigAnnoDomain + "/previousNamespaces"
```
this means more tests change, but mechanically.
This change doesn't hurt users, because these annotations are only present in the build pipeline.